data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelLogsInputCodecConfig.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.otel.codec;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.AssertTrue;
+ 
+/**
+ * Configuration class for {@link OTelLogsInputCodec}.
+ */
+public class OTelLogsInputCodecConfig {
+    static final String JSON_FORMAT = "json"; 
```

Comment:
This should be an `enum`. It will help with our auto-generated schema and documentation.

I've submitted a few PRs to correct these, such as #5025 and #5035.

Please see these:

https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/WhitespaceOption.java

https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/key-value-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/keyvalue/WhitespaceOptionTest.java

This is where it is used:

https://github.com/opensearch-project/data-prepper/blob/6e51f512e842d243102e192273e01c0aba6ad836/data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessorConfig.java#L129-L133

Reply:
Added `OTelLogsFormatOption`.